modules/installation-configuration-parameters.adoc

```diff
@@ -515,6 +518,12 @@ ifdef::ibm-power[]
 |String
 endif::ibm-power[]
 
+ifdef::azure-mixed-arch[]
+|`compute.architecture`
+|Determines the instruction set architecture of the machines in the pool. Currently, heterogeneous clusters are in technology preview. Valid values are `amd64` and `arm64`.
```

Comment:
this change is part of the installer and is targeted for 4.13. so it is not supported yet and probably has to be removed.